pkg/utils/utils.go

```diff
@@ -150,3 +151,17 @@ func ParsePKey(pKey string) (int, error) {
 
 	return int(i), nil
 }
+
+// ParseNetworkId returns the network name and network namespace
+func ParseNetworkId(networkId string) (string, string, error) {
+	separatorIndex := strings.LastIndex(networkId, ".")
+	if separatorIndex < 1 || separatorIndex == len(networkId)-1 {
+		return "", "", fmt.Errorf("invalid networkId %s, should be <networkName>.<networkNamespace>", networkId)
+	}
+	return networkId[:separatorIndex], networkId[separatorIndex+1:], nil
+}
+
+// GenerateNetworkId returns the network name and network namespace with . separation
+func GenerateNetworkId(network *v1.NetworkSelectionElement) string {
+	return fmt.Sprintf("%s.%s", network.Name, network.Namespace)
```

Comment:
can we do this the other way around ? i.e first namespace then network, seems more correct as it reflects the heirarchial nature of namespace/network better